proposals/wgs/training/wgs.yaml

```diff
@@ -0,0 +1,85 @@
+wgs:
+- dir: wg-training
+  name: Training
+  mission_statement: >
+    WG Training covers developing, deploying, and operating training jobs on Kubeflow.
+  charter_link: charter.md
+  label: area/wg-training
+  leadership:
+    chairs:
+    - github: gaocegege
+      name: Ce Gao
+      company: Caicloud
+    - github: johnugeorge
+      name: Johnu George
+      company: Cisco
+    - github: terrytangyuan
+      name: Yuan Tang
+      company: Ant Group
+    tech_leads:
+    - github: andreyvelich
+      name: Andrey Velichkevich
+      company: Cisco
+    - github: gaocegege
+      name: Ce Gao
+      company: Caicloud
+    - github: Jeffwan
+      name: Jiaxin Shan
+      company: Amazon Web Services
+    - github: johnugeorge
+      name: Johnu George
+      company: Cisco
+    - github: terrytangyuan
+      name: Yuan Tang
+      company: Ant Group
+  meetings:
+  - description: Regular WG Meeting (Asia & Europe)
+    day: Wednesday
+    time: "03:00"
+    tz: PT (Pacific Time)
+    frequency: monthly - first Wednesday every month
+    archive_url: https://docs.google.com/document/d/1K0Qfr0zTuARtAg7ETg9hsMZPh0MT2QUxqGAYvM1Vpec/edit
+  - description: Regular WG Meeting (US)
+    day: Wednesday
+    time: "10:00"
+    tz: PT (Pacific Time)
+    frequency: monthly - third Wednesday every month
+    archive_url: https://docs.google.com/document/d/1K0Qfr0zTuARtAg7ETg9hsMZPh0MT2QUxqGAYvM1Vpec/edit
```

Comment:
Does it work for you?

@terrytangyuan @andreyvelich @johnugeorge @Jeffwan

Reply:
Yes this works.

Reply:
The link to the Google Doc works. The US/Asia/Europe here just means they are more friendly, right? Maybe change them to "Regular WG Meeting (Asia & Europe friendly)"? 

Also is 10:00 PT am or pm?

Reply:
Thanks @gaocegege, yes it is working for me.

Reply:
If it's 10am PT then it works for me.

Reply:
@terrytangyuan 

I will update it. I think it should be 10 am.